mne/viz/tests/test_topomap.py

```diff
@@ -557,6 +557,19 @@ def test_plot_topomap_bads():
     plt.close('all')
 
 
+def test_plot_topomap_bads_grad():
+    """Test plotting topomap with bad gradiometer channels (gh-8802)."""
+    import matplotlib.pyplot as plt
```

Comment:
No need to nest these imports anymore due to pytest infrastructure, eventually we can move these to the top. But it can be another PR since this whole file suffers from our old conventions.